tools/schemacode/pyproject.toml

```diff
@@ -20,19 +73,43 @@ exclude = '''
   )/
 )
 '''
+include = '\.pyi?$'
+line-length = 99
+target-version = ['py39']
```

Comment:
```suggestion
```

The `target-version` is not necessary if `project.requires-python` is set.

Reply:
ah yeah true